### DISCUSSION TITLE
Problem with "This request has been blocked"

### DISCUSSION THREAD

**Discussion by matiechaza** [dedoc/scramble#310] [Q&A]
Hi, I have the following problem on my pre-production server
![Captura de pantalla 2024-02-16 a la(s) 12 02 40](https://github.com/dedoc/scramble/assets/50587234/a134ff08-549f-4453-8b89-856d711c96fd)

Did that happen to anyone else? how can it be resolved?

The specific message is: _web-components.min.js:2 Mixed Content: The page at 'https://[my-url]/docs/api#/' was loaded over HTTPS, but requested an insecure resource 'http://[my-url]/docs/api.json'. This request has been blocked; the content must be served over HTTPS._

Thanks.

**Comment by romalytvynenko** [comment]
Please make sure `api.json` is served from `https`. This can be done by enforcing `https` on app level.
